src/slices.py

```diff
@@ -12,26 +12,26 @@
 a = [2, 4, 1, 7, 9, 6]
 
 # Output the second element: 4:
-print()
+print(a[1:2])
 
 # Output the second-to-last element: 9
-print()
+print(a[4:-1])
 
 # Output the last three elements in the array: [7, 9, 6]
-print()
+print(a[3:])
 
 # Output the two middle elements in the array: [1, 7]
-print()
+print(a[2:-2])
```

Comment:
Nice. Keep in mind you could also use a dynamic 'helper variable' like: mid = len(a) // 2 in order to access the middle element(s) regardless of the size of the array. The double backslash operator rounds the result down to the nearest whole number.